Userland/Libraries/LibWeb/CSS/StyleValue.cpp

```diff
@@ -299,15 +299,8 @@ BackgroundStyleValue::BackgroundStyleValue(
     NonnullRefPtr<StyleValue> attachment,
     NonnullRefPtr<StyleValue> origin,
     NonnullRefPtr<StyleValue> clip)
-    : StyleValue(Type::Background)
-    , m_color(color)
-    , m_image(image)
-    , m_position(position)
-    , m_size(size)
-    , m_repeat(repeat)
-    , m_attachment(attachment)
-    , m_origin(origin)
-    , m_clip(clip)
+    : StyleValueWithDefaultOperators(Type::Background)
```

Comment:
The commit message has a couple of mistakes.
- "manually equality operators"
- "This also a little helper"

Reply:
Thanks! Should have proof read that!